duckduckgo/iOS
# Autofill list view changes

Task/Issue URL: https://app.asana.com/0/0/1202808746542215/f
Tech Design URL:
CC:

**Description**:
Various UI detail updates including fixing items 2.1, 2.3, 4, 5.1 and 5.3. Also additional state handling for navigation bar buttons, dynamic height support added to cells + other views, full dark mode support, minor other UI tidy ups. 

Note that layout is targeting iPhone portrait only for now, will tackle landscape + iPad support in another PR  



**Steps to test this PR**:
1. Run through the various UI states and confirm everything matches design paying special attention to SearchController and navigation buttons
2. Also run through in dark mode 



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
